desugarer.go

```diff
@@ -128,16 +128,21 @@ func desugarFields(location LocationRange, fields *astObjectFields, objLevel int
 
 	// Remove methods
 	// TODO(dcunnin): this
-	for _, field := range *fields {
+	for i := range *fields {
+		field := &((*fields)[i])
```

Comment:
B/c it's modified in place. I don't want to make a copy, b/c then the changes won't be made in the "real" ast.